plugins/Monolog/config/config.php

```diff
@@ -70,6 +70,36 @@
         return Logger::WARNING;
     }),
 
+    'log.level.file' => DI\factory(function (ContainerInterface $c) {
```

Comment:
not too important though if not trivial... maybe we could just have at least a reusable function for them as likely we won't be adding any other loggers.

Reply:
Yes, but we'd have to create a DefinitionSource like the IniConfigDefinitionSource and add it during container creation. Except this definition source would be specific to the Monolog plugin, so we'd also have to figure out how to make ContainerFactory extendable in a plugin, but it's used before plugins are loaded, etc, etc. I think it would turn very complicated pretty quickly.

Reply:
all good 👍